homeassistant/components/switch/tplink.py

```diff
@@ -78,16 +81,25 @@ def device_state_attributes(self):
 
     def update(self):
         """Update the TP-Link switch's state."""
-        self._state = self.smartplug.state
-
-        if self._emeter_present:
-            emeter_readings = self.smartplug.get_emeter_realtime()
-
-            self._emeter_params[ATTR_CURRENT_CONSUMPTION] \
-                = "%.1f W" % emeter_readings["power"]
-            self._emeter_params[ATTR_TOTAL_CONSUMPTION] \
-                = "%.2f kW" % emeter_readings["total"]
-            self._emeter_params[ATTR_VOLTAGE] \
-                = "%.2f V" % emeter_readings["voltage"]
-            self._emeter_params[ATTR_CURRENT] \
-                = "%.1f A" % emeter_readings["current"]
+
```

Comment:
Please remove this blank line. There shouldn't be any blank lines surrounding a function's docstring.